design/acme-orders-challenges-crd.md

```diff
@@ -409,7 +409,7 @@ creating resources in order to solve http01 challenges).
 * keeping the `status` field up to date with details of the challenge so that
 the Order controller can make decisions based on the state of challenges.
 
-One area to highlight, is the behaviour of the Challenge controller wrt. challenges
+One area to highlight, is the behaviour of the Challenge controller w.r.t. challenges
```

Comment:
This is partially because the only two instances in this repository were inconsistent and also because it's the right thing.